### DISCUSSION TITLE
Page 403

### DISCUSSION THREAD

**Discussion by KNA7022** [SnailyCAD/snaily-cadv4#766] [Q&A] [answered]
i cant open cad-settings page and admin page when i used other way into admin page i cant set Values

**Comment by casperiv0** [comment]
I don't know what you mean here. Are you able to provide a video reproduction?

**Reply by casperiv0** [reply]
Also, for issues open a GitHub issue. Not a discussion.

**Reply by KNA7022** [reply]
sorry My English is not very good. I will record a video

**Comment by casperiv0** [comment, marked as answer]
Answered here: https://cad-docs.caspertheghost.me/docs/installations/methods/standalone#accesing-snailycadv4
